src/Hooks.cpp

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Axel Huebl, Chad Mitchell, Eric  Stern
+ * License: BSD-3-Clause-LBNL
+ */
+#include "ImpactX.H"
+
+#include <string>
+
+
+namespace impactx
+{
+    void ImpactX::call_hook (std::string const & name)
+    {
+        if (m_hook.contains(name))
```

Comment:
This is C++20. We can use `find` or `count`:
https://en.cppreference.com/w/cpp/container/unordered_map/find.html
https://en.cppreference.com/w/cpp/container/unordered_map/count.html